frontend/public/components/utils/list-input.tsx

```diff
@@ -63,6 +64,9 @@ export class ListInput extends React.Component<ListInputProps, ListInputState> {
             </div>
           </div>
         ))}
+        <div className="help-block" id={describedBy}>
```

Comment:
```suggestion
       {helpText && <div className="help-block" id={describedBy}>{helpText}</div>}
```

Reply:
@spadgett i had this before, and thought you'd just say make help text required and we won't need this logic. And i also thought, if helptext is optional now the aria-describedBy will be provided but null.

Reply:
If you provide `undefined` for `aria-describedby` when there's no help text, the attr should be left out. It should be straightforward to handle.

Reply:
@spadgett oh perfect, yeah its not a problem i was just trying to anticipate your feedback :)